lib/jekyll/commands/new_theme.rb

```diff
@@ -22,7 +26,8 @@ def process(args, opts)
         raise Jekyll::Errors::InvalidThemeName, "You must specify a theme name."
       end
 
-      new_theme_name = args.join("_")
+      new_theme_name = args.first
+      opts["theme_path"] = args[1..-1].join(" ") if args.size > 1
```

Comment:
@envygeeks I agree with your point entirely, I was aiming for consistency with `Jekyll::Commands::New` (which uses spaces in the directory name) and I didn't fully understand this command's existing logic. @ashmaroli's example has explained it to me :smile: